.scalafix.conf

```diff
@@ -1,17 +1,22 @@
 rules = [
-  NoAutoTupling
-  RemoveUnused
+  SortImports
+  Disable
+  DisableSyntax
+  ExplicitResultTypes
   LeakingImplicitClassVal
+  MissingFinal
+  NoAutoTupling
   NoValInForComprehension
   ProcedureSyntax
-  DisableSyntax
-  ExplicitResultTypes
-  Disable
+  RemoveUnused
 ]
 
-ExplicitResultTypes {
-  unsafeShortenNames = true
-}
+# the same order as IntelliJ does it, we don't want Scalafix and IDEA fight each other
+SortImports.blocks = [
+  "java."
+  "*"
```

Comment:
I wonder why not this order:
`java`
`scala`
*the rest*

That would make much more sense to me.

Reply:
> ``` # the same order as IntelliJ does it, we don't want Scalafix and IDEA fight each other```

Reply:
This basically means that whatever IDEA decides to do we will honor it in our rules. There are people using other editors so maybe we should have rules suiting our needs and wishes.

Reply:
intellij is the most commonly used by far - so i wouldn't fight it if at all possible ... for once i agree with @sideeffffect :D

that being said i think we are really nitpicking at this point

Reply:
I'm sorry for being so stubborn about this -- it's just that I think IDEA and Scalafix not being in sync would impose huge user experience burden onto many people (me included)